Loop/Extensions/UIFont.swift

```diff
@@ -27,4 +27,16 @@ extension UIFont {
 
         self.init(descriptor: descriptor, size: 0)
     }
+
+    static func heavy(_ textStyle: UIFont.TextStyle) -> UIFont {
```

Comment:
It would be tricky to finagle, since `title1` isn't a UIFont but a UIFont.TextStyle, and implicit member expressions can't be chains. So making it a member on UIFont would look something like `UIFont.preferredFont(forTextStyle: .title1).heavy()`.

Reply:
All good, then. Just thought I'd ask. 👍